src/cmd/builtin.py

```diff
@@ -1579,3 +1581,13 @@ async def _reloadbotcommands(message, command, silent=False):
         await Msg.response(message, "Bot commands reloading is started...", silent)
         bc.commands.update(reload=True)
         await Msg.response(message, "Bot commands reloading is finished", silent)
+
+    @staticmethod
+    async def _permlevel(message, command, silent=False):
+        """Get permission level for current user
+    Usage: !permlevel"""
+        if not await Util.check_args_count(message, command, silent, min=1, max=1):
+            return
+        info = message.author
+        perm_level = bc.config.users[message.author.id].permission_level
+        await Msg.response(message, f"Permission level for {info.nick} is {perm_level}", silent)
```

Comment:
me: $permlevel
bot: Permission level for None is 2

info.nick contains value only if nickname is changed for current server according to [discord.py API specs](https://discordpy.readthedocs.io/en/latest/api.html#discord.Member.nick). I think you should take a look at this piece of code to handle it for every case: https://github.com/aobolensk/walbot/blob/de3d4104584bfe1e7b9bdf528cbec065f8db8352/src/cmd/builtin.py#L452

Reply:
Fixed.